gdal/frmts/gtiff/frmt_gtiff.html

```diff
@@ -352,6 +352,10 @@ <h3>Creation Options</h3>
 
 <li><p><b>ZSTD_LEVEL=[1-22]</b>:  Set the level of compression when using ZSTD compression. A value of 22 is best (very slow), and 1 is least compression. The default is 9.</p></li>
 
+<li><p><b>WEBP_QUALITY=[1-100]</b>:  Set the WEBP quality when using WEBP compression.  A value of 100 is best quality (least compression), and 1 is worst quality (best compression).  The default is 75.</p></li>
```

Comment:
was wondering if we couldn't use WEBP_QUALITY=100 as a special hint for lossless ? (the JP2KAK driver does that for example)

Reply:
I wondered the same thing, I just liked the boolean parameter and it followed http://www.gdal.org/frmt_webp.html

Reply:
yes, we are not always consistent between drivers in this regard. I don't have a strong opinion